cmd/src/lsif_upload.go

```diff
@@ -59,11 +59,11 @@ Examples:
 		fmt.Println(usage)
 	}
 	var (
-		repoFlag        = flagSet.String("repo", "", `The name of the repository. By default, derived from the origin remote.`)
+		repoFlag        = flagSet.String("repo", "", `The name of the repository (e.g. github.com/gorilla/mux). By default, derived from the origin remote.`)
```

Comment:
Does a URL work? If so maybe: `The name or fetch URL…`? (I _think_ anything that can be passed to `git fetch` works, but I'm not sure)

Reply:
We explicitly don't pass git://foo/bar.git (we parse it out below).